listings/ch00-introduction/if_else/src/access_control.cairo

```diff
@@ -0,0 +1,23 @@
+use starknet::ContractAddress;
+
+#[contract]
+mod SimpleAccessControl {
+    use starknet::get_caller_address;
+
+    struct Storage {
+        owner: starknet::ContractAddress, 
+    }
+
+    #[constructor]
+    fn constructor(_address: starknet::ContractAddress) {
+        owner::write(_address);
+    }
+
+    #[external]
+    fn onlyOwner() -> bool {
+        if (get_caller_address() == owner::read()) {
```

Comment:
in that case, we can just simplify it to `return get_caller_address() == owner::read()`, it's more idiomatic 😄